### ISSUE TITLE
Not Display UI or any Functionality

### ISSUE THREAD

**Issue by Darjiankit** [tprouvot/Salesforce-Inspector-reloaded#215]
**Describe the bug**
When i installed this extension , reloaded page multiple times , login logout many times but still this extension not displaying anything to me just a blank white page on the side when i click little arrow icon.

**To Reproduce**
1. Go to 'Anywhere in the org'

**Expected behavior**
i should open extension with UI

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser [chrome]
 - Version 113.0.5672.127 (Official Build) (32-bit)

<img width="261" alt="Atcual" src="https://github.com/tprouvot/Salesforce-Inspector-reloaded/assets/32826488/7d69ed79-f459-4f6b-a8ab-d34a61b81f33">
<img width="118" alt="Expected" src="https://github.com/tprouvot/Salesforce-Inspector-reloaded/assets/32826488/7d4b26ac-0464-4cb6-b0e6-85c9036a3d47">

**Comment by tprouvot** [comment]
Hi @Darjiankit ,
You need to enable third party cookies on chrome

<img width="737" alt="image" src="https://github.com/tprouvot/Salesforce-Inspector-reloaded/assets/35368290/b54c46db-7919-4b0c-914a-ba20c9d96456">

**Comment by tprouvot** [comment]
I'll close the issue, if the issue is not solved feel free to add a comment an I'll reopen it

**Comment by Darjiankit** [comment]
Ok got it thanks , i checked like from my company side its blocked so i can not make changes to it as its not editable but thanks for clarification, i checked in my personal laptop and its working

**Comment by tprouvot** [comment]
FYI the plugin is also available on Edge and Firefox

**Comment by Darjiankit** [comment]
Yes but on edge all extensions are blocked by our company for salesforce so i need to use some of extensions in chrome only
